### PR TITLE
Update email forms to post to Privacy Tips

### PR DESCRIPTION
**Pull request by jbarrett**

### CODE REVIEW

File: share/site/duckduckgo/styleguide.tx [hunk 1]
```diff
@@ -289,13 +289,15 @@
 			<hr class="hr--mid" />
 			
 			<div class="t-mid">
-				<form class="frm--newsletter" action="https://flash.duckduckgo.com/e.js" method="post" name="y">
-					<input type="hidden" name="add" value="1" />
-					<input type="text" placeholder="Enter your email address" class="frm__input" name="from" />
+				<form class="frm--newsletter" action="https://duck.co/s/a" method="post" name="y">
```
Comment (bbraithwaite) [inline share/site/duckduckgo/styleguide.tx]: I know that you're just replacing what's there, but I'm confused at to why this post is submitting at all? It's an example for the style guide...

Comment (jbarrett) [inline share/site/duckduckgo/styleguide.tx]: We could remove the form elements just as easily. Suspect it may have been a c/p job initially.

**Comment by bbraithwaite** [comment]
Tested end-to-end. 👍 Merge when ready.

**Comment by jbarrett** [comment]
Cheers, @bbraithwaite! 👍

**Pull request merged**